.ci.yaml

```diff
@@ -2978,6 +2978,7 @@ targets:
       - dev
       - beta
       - stable
+      - flutter-\d+\.\d+-candidate\.\d+
```

Comment:
FYI, you'll likely want to CP this to the beta to be able to verify this is working